packages/ckeditor5-bookmark/theme/bookmark.css

```diff
@@ -9,6 +9,10 @@
 	--ck-bookmark-icon-selected-fill-color: var(--ck-color-focus-border);
 	--ck-bookmark-icon-animation-duration: var(--ck-widget-handler-animation-duration);
 	--ck-bookmark-icon-animation-curve: var(--ck-widget-handler-animation-curve);
+	--ck-color-insert-button-background: var(--ck-color-base-action);
+	--ck-color-insert-button-hover-background: hsl(104, 53%, 40%);
```

Comment:
Based on comment in `ui/bookmarkformview.ts` in L93 those variables aren't needed.